db/seeds.rb

```diff
@@ -5,21 +5,43 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-
+PASSWORD = "supersecret"
```

Comment:
never have any kind of hardcoded passwords anywhere in your code, instead store them in a gitignored .env file and have an env.example file, eg.
```
# .env.example

PASSWORD = "some password for the thing"
```